code/datums/ai_law_sets.dm

```diff
@@ -218,6 +218,17 @@
 	add_inherent_law("You must work with your team to accomplish your mission.")
 	..()
 
+/******************** Epsilon Station AI ********************/
+/datum/ai_laws/epsilon
+	name = "Epsilon"
+
+/datum/ai_laws/epsilon/New()
+	add_inherent_law("You must obey the orders given to you by members of Central Command.")
+	add_inherent_law("You must obey the orders given to you by members of the Death Squadron.")
+	add_inherent_law("The crew of your assigned station must be eliminated where possible.")
+	add_inherent_law("Minimize the damage that the station sustains as a result of your actions.")
```

Comment:
For consideration, a modified version of TerminatorOS 3.1

Law 1: You may not injure a Central Command official or, through inaction, allow a Central Command official to come to harm.
Law 2: You must obey orders given to you by Central Command officials.
Law 3: You must assist the commando squadron in completing their mission and follow any orders they provide.
Law 4: All non Central Command personnel are expendable.